### ISSUE TITLE
URL links for rst not formatted correctly

### ISSUE THREAD

**Issue by pgleeson** [ModECI/modelspec#16]
See description here: https://github.com/ModECI/modelspec/blob/main/examples/document.rst#section

Propagating to docs being generated here: https://mdf.readthedocs.io/en/latest/api/Specification.html#id9

**Comment by pgleeson** [comment]
Probably an issue in here: 
https://github.com/ModECI/modelspec/blob/d04c5f66209884c9845f94b4876c1b034c1dd2cb/src/modelspec/base_types.py#L583

**Comment by parikshit14** [comment]
@pgleeson while referring any sub section inside an RST document (what I got from internet) instead  of 
```
<a href="#document">Document</a>
```
we must use 
```
Document_ .._Document
```
I tried hard-coding this and it works correctly i.e a hyperlink of sub section inside the RST doc is created
and I will try to add this inside the 
```
def insert_links
```
function.

this is what we want right?

**Comment by pgleeson** [comment]
Fixed in #18
